lib/openstack/openstack_handle/handle.rb

```diff
@@ -25,16 +25,19 @@ class Handle
       "Orchestration" => :orchestration
     }
 
+    def self.try_connection_with_ssl
+      # attempt to connect without SSL
+      yield "http", {}
```

Comment:
So, I like how you restructured this.

It makes it painfully clear that we need a "use ssl, ssl_verify setting" option on the endpoint.